src/Sylius/Component/Variation/Model/Option.php

```diff
@@ -24,52 +24,35 @@
 class Option extends AbstractTranslatable implements OptionInterface
 {
     /**
-     * Property id.
-     *
      * @var mixed
      */
     protected $id;
 
     /**
-     * Internal name.
-     * See description in interface.
```

Comment:
This should be removed, but line above can stay (distinguishes from `presentation`).